gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/initializer/QuickEditorInitializer.kt

```diff
@@ -0,0 +1,15 @@
+package com.gravatar.quickeditor.initializer
+
+import android.content.Context
+import androidx.startup.Initializer
+import com.gravatar.quickeditor.QuickEditorContainer
+
+internal class QuickEditorInitializer : Initializer<QuickEditorContainer> {
```

Comment:
⛏ I think it looks better with:

```suggestion
internal class QuickEditorContainerInitializer : Initializer<QuickEditorContainer> {
```

Reply:
Agree, fixed!